helm/values.schema.json

```diff
@@ -0,0 +1,1856 @@
+                            "examples": [
+                              "johndoessupersecurepassword"
+                            ]
+                          }
+                        },
+                        "required": [
+                          "username",
+                          "password"
+                        ]
+                      },
+                      {
+                        "$id": "#/properties/validators/items/anyOf/0/properties/auth/OneOf/1",
+                        "type": "object",
+                        "title": "The second OneOf schema",
+                        "properties": {
+                          "secret": {
+                            "$id": "#/properties/validators/items/anyOf/0/properties/auth/OneOf/1/properties/secret",
+                            "type": "string",
+                            "title": "The auth secret schema",
+                            "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*",
+                            "description": "Name of a Kubernetes secret containing authentication credentials.",
+                            "examples": [
+                              "my-notary-auth-creds"
+                            ]
+                          }
+                        },
+                        "required": [
+                          "secret"
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "required": [
+                  "host",
+                  "trust_roots"
+                ]
+              }
+            },
+            {
+              "$id": "#/properties/validators/items/anyOf/1",
+              "type": "object",
+              "title": "The second anyOf schema",
+              "description": "A notary V2 validator instance. This validator is not functional at this point of time and just reserved for future use.",
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "notaryv2"
+                  }
+                }
+              },
+              "then": {}
+            },
+            {
+              "$id": "#/properties/validators/items/anyOf/2",
+              "type": "object",
+              "title": "The third anyOf schema",
+              "description": "A cosign validator instance.",
+              "examples": [
+                {
+                  "trust_roots": [
+                    {
+                      "name": "mykey",
+                      "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni\nqiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==\n-----END PUBLIC KEY-----\n"
+                    }
+                  ]
+                }
+              ],
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "cosign"
```

Comment:
I have tried to violate any of the schema rules below and could not get an error for that via `helm lint`. Is that expected or something off?

Reply:
there was sth off - thanks!! It should work now (excited for the integration tests :dancers: :grin:)